things/database.py

```diff
@@ -626,7 +626,7 @@ def make_filter(column, value):
 
 def make_date_filter(date_column, offset):
     """
-    Return an SQL filter to limit the date range of the SQL query.
+    Return a SQL filter to limit the date range of the SQL query.
```

Comment:
Actually, both is correct ;)

Reply:
haha, you're right! It depends on how you pronounce it, I see it now. :-)
I think I compared "a SQL filter" and "an SQL filter" on Google and the former had more hits.